src/backend/src/form-responses/form-responses.service.ts

```diff
@@ -0,0 +1,93 @@
+import { Injectable, BadRequestException } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { FormResponse } from '../entities/formResponse.entity';
+import { User } from '../entities/user.entity';
+import { Question } from '../entities/question.entity';
+import { CreateFormResponseDto, CreateMultiFormResponsesDto } from './dto';
+
+@Injectable()
+export class FormResponsesService {
+  constructor(
+    @InjectRepository(FormResponse)
+    private formResponsesRepository: Repository<FormResponse>,
+    @InjectRepository(User)
+    private readonly usersRepository: Repository<User>,
+    @InjectRepository(Question)
+    private readonly questionsRepository: Repository<Question>,
+  ) {}
+
+  findAll(): Promise<FormResponse[]> {
```

Comment:
Is there a reason this is not async, but the others are?

Reply:
yeah they should all be async~